sdk/include/psdk/shlobj.h

```diff
@@ -2091,6 +2091,36 @@ SHBindToParent(
   _Outptr_ LPVOID *ppv,
   _Outptr_opt_ PCUITEMID_CHILD *ppidlLast);
 
+/****************************************************************************
+ * SHCreateFileExtractIcon API
+ */
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+
+#if 0 // NOTE: Even if documented on MSDN, the ANSI function never existed on Windows!
```

Comment:
The just don't include it?

Reply:
OK I will just leave the comment (as documentation so that if someone compares with MSDN or PSDK and sees the ansi version, (s)he shouldn't be surprised it doesn't exist in our header).

Reply:
Fixed.